<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelaaTilitapahtumia</class>
 <widget class="QDialog" name="SelaaTilitapahtumia">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>358</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QTextEdit" name="textSaldo"/>
       </item>
       <item>
        <widget class="QTextEdit" name="textOmistaja"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textTapahtumat"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
